webpack.config.js

```diff
@@ -0,0 +1,77 @@
+const webpack = require("webpack");
+const path = require("path");
+const CopyWebpackPlugin = require("copy-webpack-plugin");
+const ExtractTextPlugin = require("extract-text-webpack-plugin");
+const package = require("./package");
+const widgetName = package.widgetName;
+
```

Comment:
add something like this `const packageName = package.widgetName.toLowerCase();`

Reply:
I tried that before, It jam. So I've to code packageJson's widgetName to lowercase.